python/python/bystro/supervised_ppca/gf_generative_pt.py

```diff
@@ -100,6 +101,7 @@ def fit(
         X: NDArray[np.float_],
         progress_bar: bool = True,
         seed: int = 2021,
+        sherman_woodbury: bool = False,
```

Comment:
What is the downside to setting this to True (using SW by default)

Reply:
This probably will only provide large gains in performance when we switch from hundreds of covariates to tens of thousands.